ansible/roles/nginx-proxy/templates/openshift/fluentd-config.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: '{{ community_signing_pipeline_name }}-fluentd-{{ suffix }}'
+  labels:
+    io.kompose.service: "{{ community_signing_pipeline_name }}"
+    app: "{{ community_signing_pipeline_name }}"
+    suffix: '{{ suffix }}'
+    env: '{{ env }}'
+    paas.redhat.com/appcode: "{{ community_signing_pipeline_appcode }}"
+type: Opaque
+stringData:
+  fluentd.conf: |-
+    # fluentd
+    <source>
+      @type http_healthcheck
+      port 8082
+      bind 0.0.0.0
+    </source>
+    <label @FLUENT_LOG>
+      # throw those away, since they are already
+      # logged in the stdout
+      <match fluent.*>
+        @type null
+      </match>
+    </label>
+
+    # Nginx access logs
+    <source>
+      @type tail
+      tag nginx.access
+      path {{ nginx_proxy_access_log_location }}
+      pos_file /tmp/nginx_access.log.pos
+      <parse>
+        @type regexp
+        expression /^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)"(?:\s+\"?(?<http_x_forwarded_for>[^\"]*)\"?)?)?$/
+        time_format %d/%b/%Y:%H:%M:%S %z
+      </parse>
+    </source>
+
+    # Nginx error logs
+    <source>
```

Comment:
Are the format and parser correctly setup? I checked your namespace and nginx deployment and still see quite lot of `pattern not matched` errors.